templates/cargo/README.md

```diff
@@ -2,18 +2,23 @@
 
 [![Actions Status][actions badge]][actions]
 [![CodeCov][codecov badge]][codecov]
+[![Wcc][wcc badge]][wcc]
 [![LICENSE][license badge]][license]
 
 {{ name }}'s description
 
+**Note that**: the created Github Actions workflow uses a static code analysis tool, called `weighted-code-coverage`, which produces an `html` report that is hosted on [Github Pages](https://docs.github.com/en/pages). However, to make this work you need to set from the repository settings, following these [steps](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#publishing-with-a-custom-github-actions-workflow), Github Actions as source for Github Pages build and deployment.
+
```

Comment:
Change:

However, to make it work, you need to set the select box to `GitHub Actions` in the `GitHub Pages` panel contained in `Settings`, as described [here](link to steps...)